docs/contributing.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+All contributions welcome!
+
+## Bug Reporting
+If you find a bug, submit a bug report on GitHub Issues. 
+
+## Coding Style Guidelines
+The following are some guidelines on how new code should be written. Of course, there are special cases and there will be exceptions to these rules. 
+
+1. Format code in accordance with flake8 standard.
```

Comment:
are we going to add an action? or we expect people to just format on their own

Reply:
added one!